qemu/qemu.go

```diff
@@ -863,7 +876,11 @@ func (scsiCon SCSIController) QemuParams(config *Config) []string {
 	var qemuParams []string
 	var devParams []string
 
-	devParams = append(devParams, fmt.Sprintf("virtio-scsi-pci,id=%s", scsiCon.ID))
+	if runtime.GOARCH == "s390x" {
+		devParams = append(devParams, fmt.Sprintf("%s,id=%s", VirtioSCSI, scsiCon.ID))
```

Comment:
you don't need the if-else you use the same variable. Actually, the two lines are the same

Reply:
Oops oversight